### PR TITLE
disable some of the periodic ios jobs b/c flaky

### PR DESCRIPTION
**Pull request by clee2000**
As in title, also see https://github.com/pytorch/pytorch/issues/82890

### CODE REVIEW

**Review by huydhn — APPROVED** [review]

**Review by janeyx99 — APPROVED** [review]

File: .github/workflows/periodic.yml [hunk 1]
```diff
@@ -161,6 +163,7 @@ jobs:
 
   ios-12-5-1-arm64-custom-ops:
     name: ios-12-5-1-arm64-custom-ops
+    if: false # see https://github.com/pytorch/pytorch/issues/82890
```
Comment (janeyx99) [inline .github/workflows/periodic.yml:166]: Do comments work?

Comment (clee2000) [reply .github/workflows/periodic.yml:166]: as far as i can tell, this one seems fine, i added the ciflow/periodic label and they did get skipped

**Comment by malfet** [comment]
Looks like @jmdetloff found a fix, should we revert this change then?

**Pull request closed without merging**